### DISCUSSION TITLE
Fragment with no delay?

### DISCUSSION THREAD

**Discussion by 0x05E1** [GFW-knocker/gfw_resist_tls_proxy#117] [Ideas]
سلام
داشتم امتحان میکردم چقدر میشه delay رو کم کرد، دیدم با delay صفر جواب داد
روی ایرانسل و مخابرات برای من جواب داد
کسی می تونه امتحان کنه؟

**Comment by SinaHosseini7** [comment]
سلام
من با SNI کثیف تست کردم و جواب گرفتم ولی این رو باید شاره کنم که مقدار delay صفر انقدر کمه که عملا بسته یکجا فرستاده میشه و کل قضیه فرگمنت بی معنی میشه! من فکر میکنم که الان یا حجم ترافیک ورودی به GFW انقدر زیاده که کلا بیخیال combine شده یا طبق گفته @irgfw توی آخرین پست تلگرامشون در تاریخ 14 مارچ GFW توی بازه جمع آوری نمونه دیتا و آموزش دادن مدل های بلاکینگه و یه کمی شل کرده.
از نظر من بهترین گزینه دادن بازه بین 5 تا 10 میلی ثانیه به پارامتر interval که هم تاثیر کمی روی پنیگ اتصال اولیه داشته باشه و هم در واقعیت کارآمد باشه.
